ArchiSteamFarm/Bot.cs

```diff
@@ -2262,6 +2267,16 @@ public sealed class Bot : IDisposable {
 			ArchiWebHandler.OnVanityURLChanged(callback.VanityURL);
 		}
 
+		private void OnWalletUpdate(SteamUser.WalletInfoCallback callback) {
+			if (callback == null) {
+				ArchiLogger.LogNullError(nameof(callback));
+				return;
+			}
+			HasWallet = callback.HasWallet;
+			Balance = callback.Balance;
```

Comment:
If there is `Balance64` (likely `ulong`) then you should use that one instead.

Reply:
SteamKit2:
```
 //     Gets the balance of the wallet, in cents.
            public int Balance { get; }
```

Reply:
Yep, they didn't add `64` field I've seen in mine dump, this is fine, then keep it as it is.